From ca79834d2e0c738afbe605d7112cb05cf45e00a3 Mon Sep 17 00:00:00 2001
Message-ID: <ca79834d2e0c738afbe605d7112cb05cf45e00a3.1710967274.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1710967273.git.sharlatanus@gmail.com>
References: <cover.1710967273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 11 Mar 2024 14:32:52 +0000
Subject: [PATCH 32/49] gnu: python-suntime: Refresh package style.

* gnu/packages/astronomy.scm (python-suntime): Re-indent package.
[build-system]: Swap to pyproject-build-system
[description]: Concatenate small paragraphs into single one.

Change-Id: I368f042eeab2f49c6f41b2ccf0c9a69c8abfec17
---
 gnu/packages/astronomy.scm | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index db220857f5..2986079221 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4156,21 +4156,21 @@ (define-public python-suntime
   (package
     (name "python-suntime")
     (version "1.3.2")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "suntime" version))
-              (sha256
-               (base32
-                "1kyd1r6zcs0jmh5gq74adrnb1h7dfr1mzjq4k4vbngfiga8gfd28"))))
-    (build-system python-build-system)
-    (propagated-inputs (list python-dateutil))
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "suntime" version))
+       (sha256
+        (base32 "1kyd1r6zcs0jmh5gq74adrnb1h7dfr1mzjq4k4vbngfiga8gfd28"))))
+    (build-system pyproject-build-system)
+    (propagated-inputs
+     (list python-dateutil))
     (home-page "https://github.com/SatAgro/suntime")
     (synopsis "Sunset and sunrise time calculation python library")
     (description
-     "Python library doing sunrise and sunset time calculation.
-
-Takes a WGS84 (GPS) latitude/longitude as input as well as an UTC or local
-datetime object.")
+     "Python library doing sunrise and sunset time calculation.  Takes a
+WGS84 (GPS) latitude/longitude as input as well as an UTC or local datetime
+object.")
     (license license:lgpl3+)))
 
 (define-public python-synphot
-- 
2.41.0

